app/src/main/java/com/example/sharewhatyoucanproject/posts/PostsFragment.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.sharewhatyoucanproject.posts
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.sharewhatyoucanproject.databinding.FragmentPostsBinding
+
+class PostsFragment : Fragment()
+{
+    private var _binding: FragmentPostsBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done